notebooks/data_generator.py

```
@@ -0,0 +1,458 @@
+                self.df = self.df.drop(['step','oldbalanceDest','newbalanceDest' ,'isFlaggedFraud'],axis = 1)
+                self.num_rows = len(self.df) 
+                self.df.rename(columns={
+                    'type': 'Type',
+                    'amount': 'Amount',
+                    'nameOrig': 'ID Source',
+                    'oldbalanceOrg': 'Old Balance',
+                    'newbalanceOrig': 'New Balance',
+                    'nameDest': 'ID Dest',
+                    'isFraud': 'Is Fraud'
+                }, inplace=True)
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)